### PR TITLE
New Game: Super Monkey Ball 2

### PR DESCRIPTION
**Pull request by spicynun**
Adds Super Monkey Ball 2 to HintMachine.

### CODE REVIEW

File: HintMachine/Games/SuperMonkeyBall2.cs
Comment (nbrochu) [inline HintMachine/Games/SuperMonkeyBall2.cs:1]: Should probably be renamed SuperMonkeyBall2Connector.cs to follow the naming convention

**Comment by nbrochu** [comment]
You need to also commit the updated .csproj file. The connector file doesn't show in the Solution Explorer for people checking out the PR without it

**Comment by nbrochu** [comment]
The cover image asset also needs to be added to .csproj

**Comment by nbrochu** [comment]
Game connector works well on my end!

**Comment by spicynun** [comment]
I think I fixed all the issues! Let me know if there is anything else

**Pull request merged**